packages/perseus-score/src/widgets/expression/score-expression.ts

```diff
@@ -38,7 +39,8 @@ import type {
 function scoreExpression(
     userInput: PerseusExpressionUserInput,
     rubric: PerseusExpressionRubric,
-    strings: PerseusStrings,
+    // TODO: remove strings as a param for scorers
+    strings: any,
```

Comment:
Since `strings` isn't used, we could give it type `unknown` instead of `any`.
```suggestion
    strings: unknown,
```